### ISSUE TITLE
About accuracy.

### ISSUE THREAD

**Issue by suixin567** [Suprcode/mir2#325]
@Suprcode 
when player level up , the accuracy  is always in the initial state.  the hp、mp and others have been grow up ,Why is accuracy not growing?  this problem cause me always miss...
Do I have a contribution to make ？

**Comment by Suprcode** [comment]
the stats are based on the class ini file - that should tell you how much accuracy you gain.

**Comment by suixin567** [comment]
@Suprcode thans I only see the StartAccuracy ,but which is AccuracyRate when level up to grow?

**Comment by suixin567** [comment]
And I want to make sure the robot script is haven't  finished yet?  I think timed task is importent.

**Comment by suixin567** [comment]
@Suprcode  thanks , Can you give me some answers about the questions above?

**Comment by suixin567** [comment]
@Suprcode  I checked again, and I find that :
the PlayerObject.cs and UserObject.cs  are both set accuracy  use StartAccuracy  when player level up ,
this make player's accuracy allways no change . I think this is a bug.
if so , I want to make a contribution, and it's better give me a formula.

**Comment by CookieLOM** [comment]
The accuracy was never coded to change with level by design accuracy is controlled by the base stat then the relevant itemsnthe type of character can use. I think if you introduce accuracy increase per level it will make it even harder to balance just my thoughts though!

**Comment by suixin567** [comment]
@CookieLOM  thanks ,because the mir games that  I have played are all growth by level up , if not , it will always miss...

**Comment by CookieLOM** [comment]
Yeah i understand i guess you can add it by all means but there is alot of other stuff which would be more beneficial than this modification



Sent from my Samsung Galaxy smartphone.

-------- Original message --------

**Comment by Suprcode** [comment]
Accuracy increasing by level is not something which has ever been a thing of mir. Accuracy is intended to be added on the items you are wearing and spells you learn/cast.

I'm happy for you to add it - providing the default is 0 so it doesn't affect anyone.
